File: backend/src/config/schema-revision.ts
/** 应用唯一支持的持久化 Schema 基线。 */
export const SUPPORTED_SCHEMA_REVISION = "0003_task2_integrity_trace_fix" as const;

/** 拒绝让配置、迁移目标和 readiness 使用不同的兼容基线。 */
export function validateSchemaRevision(value: string): typeof SUPPORTED_SCHEMA_REVISION {
  if (value !== SUPPORTED_SCHEMA_REVISION) {
    throw new Error("only 0003_task2_integrity_trace_fix is supported");
  }
  return SUPPORTED_SCHEMA_REVISION;
}

File: backend/app/config/schema_revision.py
"""应用唯一支持的持久化 Schema 基线。"""

SUPPORTED_SCHEMA_REVISION = "0002_task2_domain_foundation"


def validate_schema_revision(value: str) -> str:
    """拒绝让配置、迁移目标和 readiness 使用不同的兼容基线。"""
    if value != SUPPORTED_SCHEMA_REVISION:
        raise ValueError("only 0002_task2_domain_foundation is supported")
    return value
